db/migrate/20170103194805_create_petition_information.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePetitionInformation < ActiveRecord::Migration
+  def change
+    create_table :petition_plugin_information do |t|
+      t.references :plugin_relation, index: true, foreign_key: { on_delete: :cascade }, null: false
+      t.datetime :deleted_at, index: true
+      t.timestamps null: false
+      t.string :call_to_action, null: false
+      t.integer :signatures_required, null: false
+      t.string :presentation, null: false
```

Comment:
I believe presentation is like a `text`. And on the edit view it should be a `textarea`.